### PR TITLE
chore: bump dependencies via renovate bot

### PR DESCRIPTION
**Pull request by Unix4ever**
Fixes: https://github.com/siderolabs/talos/pull/6914
Fixes: https://github.com/siderolabs/talos/pull/6915

### CODE REVIEW

**Comment by smira** [comment]
what about #6773?

**Comment by Unix4ever** [comment]
> what about https://github.com/siderolabs/talos/pull/6773?

Oh, that's an old one. Missed that

File: go.mod [hunk 1]
```diff
@@ -12,7 +12,7 @@ replace (
 
 	// See https://github.com/siderolabs/go-loadbalancer/pull/4
 	// `go get github.com/smira/tcpproxy@combined-fixes`, then copy pseudo-version there
-	inet.af/tcpproxy => github.com/smira/tcpproxy v0.0.0-20201015133617-de5f7797b95b
+	inet.af/tcpproxy => github.com/smira/tcpproxy v0.0.0-20200125044825-b6bb9b5b8252
```
Comment (frezbo) [inline go.mod:15]: i guess this is correct, the one renovate shows is wrong

**Review by frezbo — APPROVED** [review]

File: pkg/machinery/go.mod [hunk 2]
```diff
@@ -7,16 +7,16 @@ go 1.20
...
+	github.com/cosi-project/runtime v0.3.0-alpha.8
 	github.com/dustin/go-humanize v1.0.1
 	github.com/evanphx/json-patch v5.6.0+incompatible
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/go-version v1.6.0
 	github.com/jsimonetti/rtnetlink v1.3.1
 	github.com/mdlayher/ethtool v0.0.0-20221212131811-ba3b4bc2e02c
-	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
+	github.com/opencontainers/runtime-spec v1.1.0-rc.1.0.20230306064114-a7c7692b2a2c
```
Comment (smira) [inline pkg/machinery/go.mod:19]: let's not update this please, we had it discussed last week

File: go.mod (hunk 1 above)
Comment (Unix4ever) [reply go.mod]: yep, reverted

File: pkg/machinery/go.mod (hunk 2 above)
Comment (Unix4ever) [reply pkg/machinery/go.mod:19]: yeah, sorry forgot about it

**Review by smira — APPROVED** [review]

**Pull request merged**